### DISCUSSION TITLE
3D mesh from 2D tessellation for simulation

### DISCUSSION THREAD

**Discussion by xshang93** [neperfepx/neper#370] [Q&A] [answered]
After generating 3D meshes from a 2D tessellation with the option `-dim 3` in `neper -M`, I realized it can't be readily used for fepx simulation. Specifically, in this .msh file the node set $nset and $faset sections are missing (I think fepx relies on these to place boundary conditions?) and the orientation section is missing too. For the $nset and $faset mssing, I think the `-nset` option in module M should help, but I couldn't make it work. For the missing orientation, I believe I can just copy from tessellation file.

The commands I used are:
```
$ neper -T -n '10' -id 1 -dim 2 -domain 'square(1,1)' -ori 'random' -morpho 'gg' -reg 1 -sel 0.0018257 -o 10
$ neper -M 10.tess -cl '0.06' -order 2 -dim 3 -pl 100 -meshqualmin 1.0
```
And a picture of what I'm dealing with:
![10_iso](https://user-images.githubusercontent.com/86078353/150725401-baeee4c6-1281-427e-a87a-908ac2da47b5.png)

**Comment by xiaoruiqi** [comment]
I have encountered the same problem, is it a bug in Neper? @rquey

**Reply by rquey** [reply]
@Treue17, this has been moved to #371.

**Comment by rquey** [comment, marked as answer]
Moved to #371.
